pytest.ini

```diff
@@ -1,4 +1,7 @@
 # https://github.com/pytest-dev/pytest/issues/1978
 [pytest]
+markers =
```

Comment:
I had to run some tests one by one as otherwise those were very verbose, again I hope you do not mind to keep the marker known in case needed again in future